module/exporter-shopware-6/src/Infrastructure/Connector/SwagQLBuilder.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\ExporterShopware6\Infrastructure\Connector;
+
+/**
+ */
+class SwagQLBuilder
```

Comment:
what is SwagQLBuilder ?

Reply:
Builder to query in Shopware API. I changed to `Shopware6QueryBuilder`